index.js

```diff
@@ -1 +1,65 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db')
+const server = express();
+
+server.use(express.json())
+
+server.get('/api/users', async (req, res) => {
+    let users = await db.find();
+    if (users) {
+        res.json(users);
+    } else {
+        res.status(500).json({errorMessage: 'The users information could not be retrieved'})
+    }
+})
+
+server.get('/api/users/:id', async (req, res) => {
+    let user = await db.findById(req.params.id)
+    if (user) {
+        res.status(200).json(user)
+    } else {
+        res.status(404).json({errorMessage: 'The user with the specified ID does not exist'})
+    }
+})
+
+server.post('/api/users', async (req, res) => {
```

Comment:
for the post, you actually need to run another db method to return the actual user